gordo/machine/machine.py

```diff
@@ -133,3 +135,28 @@ def report(self):
         for reporter in map(BaseReporter.from_dict, self.runtime["reporters"]):
             logger.debug(f"Using reporter: {reporter}")
             reporter.report(self)
+
+
+class MachineEncoder(json.JSONEncoder):
+    """
+    Encode datetime.datetime objects as strings and handles any
```

Comment:
I know this is just a copy, but can we maybe fix this docstring a bit? When I read the first line I expect to read what it is, e.g. `A JSONEncoder for machine objects, handling  datetime.datetime objects as strings and handles any ...`